library/core/src/ops/range.rs

```diff
@@ -135,13 +133,11 @@ impl<Idx: PartialOrd<Idx>> Range<Idx> {
     /// The range is empty if either side is incomparable:
     ///
     /// ```
-    /// #![feature(range_is_empty)]
-    ///
     /// assert!(!(3.0..5.0).is_empty());
     /// assert!( (3.0..f32::NAN).is_empty());
     /// assert!( (f32::NAN..5.0).is_empty());
     /// ```
-    #[unstable(feature = "range_is_empty", reason = "recently added", issue = "48111")]
+    #[stable(feature = "range_is_empty", since = "1.47.0")]
```

Comment:
```suggestion
    #[stable(feature = "range_is_empty", since = "1.48.0")]
```

Reply:
It is still on track for 1.47, though barely.

Reply:
Merge to beta hasn't happened yet: https://github.com/rust-lang/rust/commits/beta

Will absolutely do that if the beta goes in before this one, @lzutao

Reply:
the milestone was changed to 1.48.

Reply:
I set it back to 1.47.